board/safety/safety_subaru.h

```diff
@@ -20,6 +20,14 @@ const SteeringLimits SUBARU_GEN2_STEERING_LIMITS = {
   .type = TorqueDriverLimited,
 };
 
+const LongitudinalLimits SUBARU_LONG_LIMITS = {
+  .min_gas = 0,
+  .max_gas = 3400,
+  .min_rpm = 0,
+  .max_rpm = 3200,
```

Comment:
is RPM safety related? can setting it high cause more acceleration (gas) than we intend? If not, then we don't need to enforce it

Reply:
Stock seems to scale throttle and rpm together, so I think its reasonable to limit it to the bounds of the stock system

Reply:
In that case, we can probably use one max value for both signals. Are they relatively close and move together?

Reply:
rpm is typically a lot less, and will slowly ramp down even when throttle goes to zero

Reply:
Do you have any stock routes showing how the EyeSight system uses it? I would understand how it uses it and how it affects acceleration. If setting both to max values results in the desired max accel, we can limit both to max_gas

Reply:
![image](https://github.com/commaai/panda/assets/9648890/a018e291-7769-4a24-946c-8e258f48e39c)

RPM and Throttle are roughly related. I think just scaling them together will be sufficient for the initial PR, and I'll limit RPM to the same as throttle to make it easier.